librz/analysis/fcn.c

```diff
@@ -662,6 +676,8 @@ static int fcn_recurse(RzAnalysis *analysis, RzAnalysisFunction *fcn, ut64 addr,
 	}
 
 	while (addrbytes * idx < maxlen) {
+		ut32 at_delta;
+		ut64 at;
```

Comment:
Move these 2 declaration outside the while (with a default value set) or at line 689.